premarum-backend/PreEnrollmentMgmt.Core/Entities/Output/CourseParserOutput.cs

```diff
@@ -0,0 +1,15 @@
+using PreEnrollmentMgmt.CourseParserLib;
+
+namespace PreEnrollmentMgmt.Core.Entities.Output;
+
+public class CourseParserOutput
```

Comment:
SHould we leave this in the Course Parser Lib package since no one in core is using this?